package.json

```diff
@@ -7,6 +7,10 @@
     "prepare": "husky install"
   },
   "devDependencies": {
+    "@types/express": "^4.17.20",
+    "@types/node": "^20.8.7",
```

Comment:
Remove @types/express and @types/node. Those go in each exercise, not in the root.